packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -151,3 +142,31 @@ export const generateTestDataForPage = async (
     });
   });
 };
+
+const addCommonArgs = (
+  featuresConfig: FeaturesConfig,
+  featureName: string,
+  locale: string
+) => {
+  // We need to specially handle the JSON when running on windows due to an existing bug/behavior in Powershell where inner quotes are
+  // stripped from strings when passed to a third-party program. Read more: https://stackoverflow.com/questions/52822984/powershell-best-way-to-escape-double-quotes-in-string-passed-to-an-external-pro.
+  let jsonConfig;
+  if (process.platform == "win32") {
+    jsonConfig = `"${JSON.stringify(featuresConfig).replaceAll(`"`, `\\"`)}"`;
```

Comment:
just to double check, does this correctly handle cases where an inner string already has an escaped quote? So like if my json string looks like `{"msg": "Hello \"friend\""}` that will parse correctly? It presumably would become `\\\"` but I think it might need to be `\\\\"` because the first slash isn't being doubled (the double slash in general is odd to me though).

Reply:
This doesn't work but it also doesn't work currently anyways: `Stream validation diagnostic for stream "my-stream\"-id-1": invalid name my-stream"-id-1 at position: /source`

Reply:
there's a change I can make to the regex to make it more robust which I'll add.